Sources/Fuzzilli/Evaluation/ProgramCoverageEvaluator.swift

```diff
@@ -71,24 +103,29 @@ public class ProgramCoverageEvaluator: ComponentBase, ProgramEvaluator {
         }
         
         let _ = fuzzer.execute(Program())
-        libcoverage.cov_finish_initialization(&context)
+        if self.shouldTrackEdges {
```

Comment:
Nit: can this become `libcoverage.cov_finish_initialization(&context, shouldTrackEdges ? 1 : 0)` ?